emails/login/html.pug

```diff
@@ -0,0 +1,105 @@
+doctype html
+include ../includes/head
+body
```

Comment:
Sorry, dass ich das jetzt erst frage, aber was machen diese PUG-Datein genau und wie nutzt du diese in der API?

Reply:
Die PUG-Dateien sind die Email-Templates, die versendet werden, wenn man sich verifizieren muss oder sein Passwort zurücksetzt.